test/docfx.E2E.Tests/DocfxSeedSiteFixture.cs

```diff
@@ -48,14 +49,16 @@ public DocfxSeedSiteFixture()
                         ContentTypeProvider = contentTypeProvider
                     }
                 };
-                Url = $"{RootUrl}:{port}";
                 WebApp.Start(Url, builder => builder.UseFileServer(fileServerOptions));
             }
             catch (System.Reflection.TargetInvocationException)
             {
                 Console.WriteLine($"Error serving \"{folder}\" on \"{Url}\", check if the port is already being in use.");
             }
-
+            catch (Exception e)
+            {
+                Console.WriteLine($"Error serving \"{folder}\" on \"{Url}\": {e}");
```

Comment:
I think whole e containing calling stack and inner exception is more clear here.